config/routes.rb

```diff
@@ -84,6 +84,14 @@
   match '/admin/newsletter' => 'admin#newsletter', :as => :admin_newsletter
   match '/admin/:action' => 'admin#:action'
 
-
+  namespace :api, :defaults => {:format => :json}  do
+    namespace :v1 do
+      resources :crops
+      resources :gardens
```

Comment:
No, let's just do crops for now. Please remove the extraneous routes; we'll do them one at a time.